### DISCUSSION TITLE
Restricting repository creation.

### DISCUSSION THREAD

**Discussion by avinash371** [community/community#35304] [Enterprise]
If we add a member to our GitHub, do we also restrict him from creating repositories in his personal profile?

**Comment by radar-blip** [comment]
I dont think you can restrict a member from doing anything in his personal space/profile.

**Reply by avinash371** [reply]
Ok, why I'm asking is he will use our account for his personal work and we have to pay for that if we are using enterprise..!!

**Reply by radar-blip** [reply]
Yes but what he does on his own account wont matter either way.

**Reply by avinash371** [reply]
That means we no need to pay for enterprise features like his actions minutes and package storage on his personal profile. That means we need to pay GitHub for the features used in organizations only ..!!
